src/full/Agda/Interaction/Response.hs

```diff
@@ -1,219 +1,30 @@
-------------------------------------------------------------------------
--- | Data type for all interactive responses
-------------------------------------------------------------------------
 
 module Agda.Interaction.Response
-  ( Response (..)
-  , RemoveTokenBasedHighlighting (..)
-  , MakeCaseVariant (..)
-  , DisplayInfo (..)
-  , GoalDisplayInfo(..)
-  , Goals
-  , WarningsAndNonFatalErrors
-  , Info_Error(..)
-  , GoalTypeAux(..)
-  , ResponseContextEntry(..)
-  , Status (..)
-  , GiveResult (..)
-  , InteractionOutputCallback
-  , defaultInteractionOutputCallback
-  ) where
+    (   module Agda.Interaction.Response
+    ,   module Agda.Interaction.Response.Boot
+    ,   WarningsAndNonFatalErrors
+    ,   InteractionOutputCallback
+    ,   defaultInteractionOutputCallback
+    )
+    where
```

Comment:
A similar concern applies here as for `Agda.Interaction.Base`.

Reply:
Renamed to `.Types` in 8fecfbf, since it contains only type definitions, plus instances & functions needed for their `deriving` clauses.

Reply:
Here are some easy options to reduce the size of `Agda.Interaction.Response.Boot`:

* Move unnecessary definitions out of `Agda.Interaction.Response.Boot` (trims max 42 lines)
* Rename `Agda.Interaction.Response.Boot` to `Agda.Interaction.Response.Base`
* Rename `Agda.Interaction.Response.Boot` to `Agda.Interaction.Response`, after renaming `Agda.Interaction.Response` to some new module `Agda.Interaction.XXX`
  (I am not sure what `XXX` should be)

Do you have a preference?

Reply:
Chose renaming to `Agda.Interaction.Response.Base`